### PR TITLE
Added nic binding method to ui docs

### PR DESCRIPTION
**Pull request by jelkosz**

### CODE REVIEW

**Comment by jelkosz** [comment]
@pcbailey please review

File: web-interface/managing-nics-modules/vm-create-nic-web.adoc [hunk 1]
```diff
@@ -9,5 +9,5 @@ Create and attach additional NICs to a virtual machine from the web console.
 . Select a virtual machine.
 . Click *Network Interfaces* to display the NICs already attached to the virtual machine.
 . Click *Create NIC* to create a new slot in the list.
-. Fill in the *NAME*, *NETWORK*, and *MAC ADDRESS* xref:networking-wizard-fields-web[details] for the new NIC.
-. Click the &#10003; button to save and attach the NIC to the virtual machine.
\ No newline at end of file
+. Fill in the *NAME*, *NETWORK*, *MAC ADDRESS* and *BINDING METHOD* xref:networking-wizard-fields-web[details] for the new NIC.
```
Comment (pcbailey) [inline web-interface/managing-nics-modules/vm-create-nic-web.adoc]: nitpick: Add a comma after "MAC ADDRESS".

File: web-interface/resource-modules/networking-wizard-fields-web.adoc [hunk 2]
```diff
@@ -15,6 +15,9 @@
 |NETWORK CONFIGURATION
 |List of available *NetworkAttachmentDefinition* objects.
 
+|BINDING METHOD
+|The definition of the link:../creating-virtual-machines/interfaces-and-networks.html#frontend[binding method]. For pod networking, the possible values are `masquerade`, `bridge` and `sriov`. For the other networks the possible values are `bridge` and `sriov`.
```
Comment (pcbailey) [inline web-interface/resource-modules/networking-wizard-fields-web.adoc]: To remain consistent with the wording of the other drop-down descriptions, I'd go with "List of available binding methods."

nitpick: Add a comma after "bridge".

File: web-interface/managing-nics-modules/vm-create-nic-web.adoc (hunk 1 above)
Comment (jelkosz) [reply web-interface/managing-nics-modules/vm-create-nic-web.adoc]: done

File: web-interface/resource-modules/networking-wizard-fields-web.adoc (hunk 2 above)
Comment (jelkosz) [reply web-interface/resource-modules/networking-wizard-fields-web.adoc]: done

Comment (apinnick) [reply web-interface/resource-modules/networking-wizard-fields-web.adoc]: Would it be appropriate to include links to documents that explain these binding methods?

**Pull request merged**